backend/models/notification.js

```diff
@@ -242,11 +309,56 @@ module.exports = {
 		return notifications;
 	},
 
+	removeClosedNotifications : async function(teamSpace, modelId, issue) {
```

Comment:
There's so much duplicated code with `upsertIssueClosedNotifications` that the shared logic really should be pushed out to separate functions.